resources/sql/application-queries.sql

```diff
@@ -650,6 +650,13 @@ SELECT
 FROM application_hakukohde_reviews
 WHERE application_key = :application_key;
 
+-- name: yesql-get-payment-obligation-for-applications
+SELECT DISTINCT ON (application_key, requirement) application_key, hakukohde, requirement, state
```

Comment:
Pitäis varmaan olla `DISTINCT ON (application_key, hakukohde)` jos tarkotuksena tuottaa rivi per hakukohde.
`requirement` saraketta ei oo syytä projisoida, kun se on aina `payment-obligation`.

Reply:
Totta